sys/ssp/Makefile.include

```diff
@@ -1,3 +1,6 @@
+# Canary value is randomized on each new build
```

Comment:
Can we move this to ` sys/ssp/Makefile`? No need to export it globally.

Reply:
I was under the impression that it is not possible to modify `CFLAGS` in `sys/ssp/Makefile`.

Reply:
Ok, did some quick tests. It seems that if the `-DSTACK_CHK_GUARD` flag is not added globally, the define won't be added to `riotbuild.h` and therefore all modules would not be rebuild if it changes.

Reply:
I think the best solution here would be only recompiling the `ssp` module on each build. Unfortunately, a change of module specific defines passed by adding a `-D` flag to `CFLAGS` in `sys/ssp/Makefile` does not result in a rebuild of the affected module.